config/pipelines.yml

```diff
@@ -0,0 +1,8 @@
+# - pipeline.id: test
+#   pipeline.workers: 1
+#   pipeline.batch.size: 1
+#   config.string: "input { generator {} } filter { sleep { time => 1 } } output { stdout { codec => plain { format => '1' } } }"
+# - pipeline.id: another_test
+#   pipeline.workers: 2
+#   pipeline.batch.size: 1
+#   config.string: "input { generator {} } filter { sleep { time => 1 } } output { stdout { codec => dots } }"
```

Comment:
I am sure that we will list all the options in the documentation that the user can change, but I think we should add the list from https://github.com/elastic/logstash/pull/6525/files#diff-37d77f4017277cc3a8ad8d054cffb4c5R8 with default values here and make it commented.